components/modal/modal.js

```diff
@@ -0,0 +1,21 @@
+const openModal = document.querySelector('#open-modal');
+const modal = document.querySelector('#modal');
+closeBtnTop = document.querySelector('#close');
+closeBtnBottom = document.querySelector('#close-button')
```

Comment:
use ; if you are using it everywhere, just for consistency of code.

Reply:
okay , I will fix this